concepticondata/conceptlists.tsv

```diff
@@ -317,3 +317,4 @@ Lundgren-2020-275	Lundgren, Olof	2020		275	areal	English	The Tupí–Guaraní la
 Ruhlen-1998-36	Ruhlen, Merritt	1998		36	basic, areal	English	Yeniseian and Na-Dene	https://www.pnas.org/content/pnas/95/23/13994.full.pdf	Ruhlen1998		The basic concepts in this list serve to compare Proto-Yeniseian and Proto-Na-Dene cognates which are not widely shared by other language families. The author argues that this large amount of cognates of basic vocabulary can only be explained by a common ancestor language.	13995-13996	
 Lee-2015-246	Lee, Sean	2015		246	basic, areal	English	Koreanic	https://doi.org/10.1371/journal.pone.0128448	Lee2015		The basic concepts in this list are selected to &quot;quantitatively describe the evolutionary characteristics&quot; (of Koreanic).	e0128448	
 Urban-2016-160	Urban, Matthias	2016		160	basic	English	Global		Urban2016	Urban2016	This list of 160 nominal items was compiled in order to compare morphological complexity across 78 languages of different language families. The concepts were organized into four semantic categories relating to nature, artifacts, the body and temporal concepts (plus some miscellaneous words). The list is not directly based on another concept list, although [Brown 1999](:bib:Brown1999) and [Buck 1949](:ref:Buck-1949-1110) provided some inspiration.	571-572	
+Koelle-1854-335	Koelle, Sigismund W.	1854		335	basic, areal	English	African languages	http://reflex.cnrs.fr/	Koelle1854		This list of lexical and phrasal items reflects the comparative vocabulary collected for more than a hundred African languages in the 19th century the Church Missionary House. It includes basic vocabulary, cultural and areal items.		Polyglotta Africana
```

Comment:
"in the 19th century **by** the Church Missionary House"
"cultural**,** and areal items."